packages/zosmf/src/api/ZosmfSession.ts

```diff
@@ -144,21 +193,33 @@ export class ZosmfSession {
     /**
      * Given command line arguments, create a REST Client Session.
      * @static
+     * @deprecated Use ZosmfSession.createSessCfgFromArgs & others
      * @param {IProfile} args - The arguments specified by the user
      * @returns {Session} - A session for usage in the z/OSMF REST Client
      */
     public static createBasicZosmfSessionFromArguments(args: ICommandArguments): Session {
         this.log.debug("Creating a z/OSMF session from arguments");
-        return new Session({
-            type: "basic",
+
+        const sessionConfig: ISession = {
             hostname: args.host,
             port: args.port,
-            user: args.user,
-            password: args.password,
-            base64EncodedAuth: args.auth,
             rejectUnauthorized: args.rejectUnauthorized,
             basePath: args.basePath
-        });
+        };
+
+        this.log.debug("Using basic authentication");
```

Comment:
Would we want this in an if statement so we are not adding both the "Using basic authentication" and "Using token authentication" messages to the debug log?

Reply:
I added an if test.